configs/hifi_flye.config

```diff
@@ -0,0 +1,6 @@
+params {
+    assembler = 'flye'
```

Comment:
I feel like I'm missing something with this set of profiles. Only one of these can be run at a time right? Should this profile just be a workflow parameter?

Reply:
There's a toy example [here](https://github.com/mahesh-panchal/Nextflow-metadata-practices) of how to implement profiles for workflows. This way has the benefit that it could be extended to support running multiple profiles, unlike nextflow configuration profiles which would overwrite each other.

Reply:
I will look into this, it would be nice to be able to run multiple profiles at the same time.